install.sh

```diff
@@ -282,8 +303,12 @@ main() {
 		--version=*)
 			VERSION="$(parse_arg "$@")"
 			;;
-		--edge)
-			EDGE=1
+		# Support edge for backwards compatibility.
```

Comment:
nit: isn't it `backward` (no -s)?

Reply:
Both forms seem to be used, but perhaps “backward” is the more common one.